wiki/en/Running-a-Server.md

```diff
@@ -136,63 +116,62 @@ Note also that mean ADSL2 transfer rate is 10 Mbit/s for downstream and 1 Mbit/s
 
-### Genre
-
-Because there is a limit of 150 servers per Directory, you need to select which Directory Server you want to advertise your server on. Select a genre if you intend to limit players to that style (players can then see your server in their genre list). This will make your server appear in the relevant server list for clients.
-
+**Custom**: This sets your Server to be a Directory on which other Servers can be listed. Note that when setting this mode, you may want also to specify the location of Server list file (See "Options" below) unless you want to disconnect all listed Servers when restarting Jamulus.
 
 ### My Server Info
 
-Type in the name, city and country so that other users can easily identify you.
+When running as a registered Server this displays the Server's name, city and country so that other users can easily identify it in the Directory listing.
 
 ### Chat Welcome Message
 
-The text entered here appears to all users when they join the server (the chat window will open automatically for them).
+The text entered here appears to all users when they join the Server (the chat window will open automatically for them). HTML is also supported.
 
 ## Options
 
 
-<figure><img src="{% include img/en-screenshots/server-window-options.inc %}" style="width:80%; border:5px solid grey;" loading="lazy" alt="Image of Jamulus server window options"></figure>
+<figure><img src="{% include img/en-screenshots/server-window-options.inc %}" style="width:80%; border:5px solid grey;" loading="lazy" alt="Image of Jamulus Server window options"></figure>
 
 ### Recording Directory
 
-This sets the path to where server's recording will be stored. With this path set, the "Enable Jam Recoder" function in the Server Setup tab will make recording start once the first person connects to the server, and stops when the last person leaves. Use the "New Recording" button to create a new sub-directory in which the recordings will be stored from then on. Note that Recordings are per track in [Audacity](https://www.audacityteam.org/) `.lof` format and [REAPER](https://en.wikipedia.org/wiki/REAPER) `.rpp`. Open the respective files to listen to them in those applications.
+This sets the path to where Server's recordings will be stored. With this path set, the "Enable Jam Recoder" function in the Server Setup tab will make recording start once the first person connects to the Server, and stops when the last person leaves. Use the "New Recording" button to create a new sub-directory in which the recordings will be stored from then on. Note that Recordings are per track in [Audacity](https://www.audacityteam.org/) `.lof` format and [REAPER](https://en.wikipedia.org/wiki/REAPER) `.rpp`. Open the respective files to listen to them in those applications.
```

Comment:
This sets the path to where **the** Server's recording... (or Servers' recordings)